### PR TITLE
Migrate more uts

### PR DESCRIPTION
**Pull request by shargon**

### CODE REVIEW

**Review by Jim8y — COMMENTED** [review]
why this change?

**Comment by shargon** [comment]
> why this change?

The smart contract or which one?

**Comment by shargon** [comment]
@Jim8y if is the SmartContract change, there are some UT that send the StackItem type as an argument, We can change the unit test, but I think that is good to be able to do that

File: tests/Neo.Compiler.CSharp.TestContracts/Contract_NULL.cs [hunk 1]
```diff
@@ -5,27 +5,27 @@ namespace Neo.Compiler.CSharp.TestContracts
 {
     public class Contract_NULL : SmartContract.Framework.SmartContract
     {
-        public static bool IsNull(byte[]? value)
+        public static bool IsNull(object? value)
```
Comment (Jim8y) [inline tests/Neo.Compiler.CSharp.TestContracts/Contract_NULL.cs:8]: I mean this change

**Review by Jim8y — APPROVED** [review]

File: tests/Neo.Compiler.CSharp.TestContracts/Contract_NULL.cs (hunk 1 above)
Comment (shargon) [reply tests/Neo.Compiler.CSharp.TestContracts/Contract_NULL.cs:8]: It checked that the object is null, but it receive a buffer, we can revert it if you prefer

Comment (Jim8y) [reply tests/Neo.Compiler.CSharp.TestContracts/Contract_NULL.cs:8]: i dont like non related change. but its ok.

**Pull request merged**